### DISCUSSION TITLE
Issues with both web app and desktop app

### DISCUSSION THREAD

**Discussion by mp377674** [soundshed/soundshed-app#53] [General]
On web app, i am able to connect to my amp, but the search bar for the tones tab is totally useless. Nothing happens

On the desktop app, the tones tab works well, but I can't get my amp to come up when i scan for devices

If i could somehow combine the two into one it would be perfect, but very frustrating as is

**Comment by coachjdykes** [comment]
The desktop app will not let me sign in

**Reply by webprofusion-chrisc** [reply]
There is no longer a Sign In feature, it's just that the desktop version of the app hasn't been updated to show that, this is blocked by a problem building the mac version.

**Comment by Keighnine** [comment]
For some unknown reason, shoundshed will not locate my Spark 40 amp so I can use and/or modify tones.  What's up with that?  I'm using a windows 10 toshiba laptop

**Reply by webprofusion-chrisc** [reply]
Sorry, no idea.  Make sure you pair the Spark with windows bluetooth first (it should pair as a bluetooth speaker). Does it work if you use Google Chrome or Microsoft Edge and try the web version https://app.soundshed.com ?

**Comment by webprofusion-chrisc** [comment]
There is a new release of the app v1.0.29 which uses a different method to connect to bluetooth and has a newer version of electron.
